### ISSUE TITLE
Credentials not used from .npmrc

### ISSUE THREAD

**Issue by Cronkan** [vscode-contrib/vscode-versionlens#185]
When i try to fetch versions from my custom npm-repo the console says this:
```
Error: NPM: parseNpmArguments Error: Unauthorized You must log in to perform this action.: 
@angular/animations    
at c:\Users\*******\.vscode\extensions\pflannery.vscode-versionlens-0.25.0\out\extension-bundle.js:1383:19   
at processTicksAndRejections (internal/process/task_queues.js:85:5)    
at async Promise.all (index 0)
```
Running `npm install` in console works.
Have run `npm config set registry http://{my-registry}` and `npm login` with credentials
My .npmrc in `c:\users\****\.npmrc`:
```
registry=http://REDACTED
//REDACTED/:_password=REDACTED
//REDACTED/:username=REDACTED
//REDACTED/:email=REDACTED
//REDACTED/:always-auth=false
```
VSCode 1.44.2
Versionlens 0.25.0

**Comment by pflannery** [comment]
@Cronkan This should be resolved in v0.26.1. Let me know how it goes. thanks

**Comment by Cronkan** [comment]
@pflannery It works now, thank you! :)
